package.json

```diff
@@ -16,8 +16,10 @@
     "url": "https://github.com/Microsoft/fast-dna/issues/new/choose"
   },
   "scripts": {
-    "docs:build": "node build/documentation/copy-package-readme.js",
-    "docs:dry-run": "node build/documentation/copy-package-readme.js --dry-run",
+    "docs:build": "node build/documentation/generate-typedocs.js",
```

Comment:
@jkmdev @awentzel my assumption is that build should do the full build for all documentation. Is that reflective of this update?

Reply:
In terms of the package readme files and typedoc api docs (Marketing and Technical documentation), that line will generate both those things. I'm fairly sure that's everything, but it might be best to see what awenzel says just in case.